Lombiq.HelpfulLibraries.OrchardCore/Data/QueryExtensions.cs

```diff
@@ -195,4 +196,4 @@ public static Task<GetPageAndPagerViewModel<T>> GetPageAndPagerAsync<T>(
     }
 }
 
-public record GetPageAndPagerViewModel<T>(IList<T> Items, IShape Pager, int Total, int PageSize, int PageIndex);
+public record GetPageAndPagerViewModel<T>(IList<T> Items, [property: JsonIgnore] IShape Pager, int Total, int PageSize, int PageIndex);
```

Comment:
Very cool, I didn't know you can do that! But now it's longer than 120 and has different meta levels which is less readable. If we break it up to multiple lines which one do you think is better?

```suggestion
public record GetPageAndPagerViewModel<T>(
    IList<T> Items,
    [property: JsonIgnore]
    IShape Pager,
    int Total,
    int PageSize,
    int PageIndex);
```

```suggestion
public record GetPageAndPagerViewModel<T>(
    IList<T> Items,
    [property: JsonIgnore] IShape Pager,
    int Total,
    int PageSize,
    int PageIndex);
```

Reply:
I think the second options more first look self-explanatory.